spec/language-python-spec.coffee

```diff
@@ -6,7 +6,7 @@ describe 'Python settings', ->
 
   beforeEach ->
     waitsForPromise ->
-      atom.workspace.open('sample.py').then (o) ->
+      atom.workspace.open('../../sample.py').then (o) ->
```

Comment:
This can actually just be `atom.workspace.open()`.  No need to specify a filename.